backend/src/main/scala/cromwell/backend/WorkflowBackendActor.scala

```diff
@@ -37,29 +42,35 @@ object WorkflowBackendActor {
 trait WorkflowBackendActor extends Actor with ActorLogging {
 
   protected implicit def ec: ExecutionContext
+  protected def calls: Seq[Call]
```

Comment:
I think it will be worthy to have some description of why calls are being added so the backend developer knows about that.